common/build.gradle

```diff
@@ -356,18 +326,10 @@ tasks.configureEach { task ->
     }
 
     if (task.name == 'assembleDebug' || task.name == 'assembleRelease') {
-        task.dependsOn 'lint', 'sourcesJar', 'pmd'
-    }
-    if (task.name == 'assembleDistRelease') {
-        task.dependsOn 'sourcesJar'
+        task.dependsOn 'lint', 'pmd'
     }
 }
 
-// This is used to generate the pom file for publishing to external maven in maven-release-jobs.yml
-tasks.withType(GenerateMavenPom).all {
-    destination = layout.buildDirectory.file("poms/${project.name}-${project.version}.pom").get().asFile
```

Comment:
Let's ensure that we are still able to run the maven release jobs with these changes
As those expect the pom file in certain location/path